app/controllers/api/messaging.php

```diff
@@ -1726,8 +2020,11 @@
         $subscribers = batch(\array_map(function (Document $subscriber) use ($dbForProject) {
             return function () use ($subscriber, $dbForProject) {
                 $user = Authorization::skip(fn () => $dbForProject->getDocument('users', $subscriber->getAttribute('userId')));
+                $target = Authorization::skip(fn () => $dbForProject->getDocument('targets', $subscriber->getAttribute('targetId')));
 
                 return $subscriber
+                    ->setAttribute('identifier', $target->getAttribute('identifier'))
```

Comment:
Let's just add the entire `target` as an attribute here since we'll need the `name` too, and that way any future attributes we need are automatically added

Reply:
done this